Misc/NEWS

```diff
@@ -278,6 +278,10 @@ Extension Modules
 Library
 -------
 
+- bpo-26187: Test that sqlite trace callback is not called multiple
+  times when schema is changing. Fixed by bpo-9303
```

Comment:
Perhaps we can say something like "Indirectly fixed by switching to use sqlite3_prepare_v2() in bpo-9303."?